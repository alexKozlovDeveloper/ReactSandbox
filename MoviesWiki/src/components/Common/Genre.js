import React, { Component } from 'react';

import styles from '../../styles/Genre.css';

const Genre = ({ genre }) => <div className={styles.genre}>
        {genre}
    </div>;

export default Genre;
