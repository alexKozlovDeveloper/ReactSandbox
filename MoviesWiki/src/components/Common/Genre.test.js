import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Genre from './Genre';

configure({ adapter: new Adapter() });

test('Genre rendering test', () => {
  const genre = 'someGenre';

  const component = shallow(<Genre genre={genre} />);
  expect(shallowToJson(component)).toMatchSnapshot();
});
