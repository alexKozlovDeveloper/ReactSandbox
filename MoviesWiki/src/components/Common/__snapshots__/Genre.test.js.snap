// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Genre rendering test 1`] = `
<div
  className="genre"
>
  someGenre
</div>
`;
